.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -115,7 +115,7 @@ ort_test_dir_utils.create_test_dir(model_path, 'temp/examples', 'test2', name_in
 ```
 
 ### More details
-https://github.com/microsoft/onnxruntime/blob/master/tools/python/PythonTools.md
+https://github.com/microsoft/onnxruntime/blob/main/tools/python/PythonTools.md
```

Comment:
```suggestion
https://github.com/microsoft/onnxruntime/blob/master/tools/python/PythonTools.md
```
onnxruntime is still using master today. I think only onnx/onnx and onnx/models need to be updated to main. Other repo links changed by this PR need to be checked whether they use main branch. Otherwise, the links will be broken.

Reply:
Only update link in onnx/model now.
@jcwchen , do you still keep the master branch since the external projects may still have the link to master branch?

Reply:
I didn't keep it. According to [here](https://github.com/github/renaming#renaming-existing-branches), the old link with master should be still workable. I don't know how long it will last though.